akka-stream/src/main/scala/akka/stream/Graph.scala

```diff
@@ -20,7 +20,7 @@ trait Graph[+S <: Shape, +M] {
    *
    * Every materializable element must be backed by a stream layout module
    */
-  private[stream] def module: StreamLayout.Module
+  def module: StreamLayout.Module
```

Comment:
Hm, this is just too dangerous. Maybe create instead an object `ModuleExtractor` that reads this value instead of making it public on Graph?